ecs-init/config/common_test.go

```diff
@@ -47,22 +48,38 @@ func TestDockerUnixSocketWithDockerHost(t *testing.T) {
 	}
 }
 
-func TestIsSupportedRegion(t *testing.T) {
-	var cases = []struct {
-		region         string
-		expectedResult bool
+func TestGetAgentBucketRegion(t *testing.T) {
+	testCases := []struct {
+		region      string
+		destination string
+		err         error
 	}{
-		{DefaultRegionName, true},
-		{"cn-north-1", true},
-		{"missing-region", false},
+		{
+			region:      "us-west-2",
+			destination: "us-east-1",
+			err:         nil,
+		}, {
+			region:      "us-gov-west-1",
+			destination: "us-gov-west-1",
+		}, {
+			region:      "cn-north-1",
+			destination: "cn-north-1",
+		}, {
+			region:      "invalid",
+			destination: "",
```

Comment:
I think we could also then use the err field to assert there should be an error for this testcase, or we could get rid of the field entirely.